src/components/LoanCollections/KivaClassicLoanCategorySelector.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<div
+		class="tw-flex tw-justify-center tw-max-w-[700px] tw-flex-wrap tw-mx-auto"
```

Comment:
Since this does't require any differences between breakpoints, can you move the `tw-max-w-700` into an inline style, to keep the class from being added to the global stylesheet.